tests/Validation/ValidationValidatorTest.php

```diff
@@ -5030,6 +5030,85 @@ public function message()
         $this->assertSame('name must be taylor', $v->errors()->get('name')[0]);
         $this->assertSame('name must be a first name', $v->errors()->get('name')[1]);
         $this->assertSame('validation.string', $v->errors()->get('name')[2]);
+
+        // Test messagebag return from validator
+
+        $v = new Validator(
+            $this->getIlluminateArrayTranslator(),
+            ['form' => ['name' => 'taylor']],
+            [
+                'form' => [
+                    new class implements Rule {
+                        private $validator;
+
+                        public function passes($attribute, $value)
+                        {
+                            $ArrayTranslator = new Translator(
```

Comment:
```suggestion
                            $arrayTranslator = new Translator(
```

Reply:
Some more of these further down in the PR.

Reply:
Good catch, will update them now!